### DISCUSSION TITLE
Customizing RSS feed

### DISCUSSION THREAD

**Discussion by joegullo** [automadcms/discussions#48] [Q&A]
Is there way to restrict what is pulled into the RSS feed? Right now, it is showing all of the pages. I would like to limit based on URL or exclude certain pages.

**Comment by marcantondahmen** [comment]
Hi, at the moment that is not possible. But I can imagine to add that feature sooner or later ...

**Reply by joegullo** [reply]
Awesome! It would also be amazing for some more blog-focused features!
